lms/templates/courseware/progress.html

```diff
@@ -38,7 +38,7 @@
   <div class="profile-wrapper">
 
     <div class="course-info" id="course-info-progress" aria-label="${_('Course Progress')}">
-      % if staff_access and studio_url is not None:
+      % if has_access_on_students_profiles and studio_url is not None:
         <div class="wrap-instructor-info">
           <a class="instructor-info-action studio-view" href="${studio_url | h}">${_("View Grading in studio")}</a>
         </div>
```

Comment:
Wait a second. I didn't realize that `staff_access` was only controlling the appearance of the "View Grading in studio" button? Coaches don't need and shouldn't have access to that. 

I don't think you need to change https://github.com/edx/edx-platform/pull/11124/files#diff-54c933c75934e410ed718cbb2345158aL41 or https://github.com/edx/edx-platform/pull/11124/files#diff-d8e9b05acd3b7dba84709ae01b99053cL988